.circleci/config.yml

```diff
@@ -37,7 +37,14 @@ jobs:
             source dev_env.sh
             source /usr/local/share/virtualenvs/tap-tester/bin/activate
             cd /root/project/
-            run-test --tap=tap-sftp tests
+            run-test --tap=tap-sftp \
+                     --target=target-stitch \
+                     --orchestrator=stitch-orchestrator \
+                     --email=harrison+sandboxtest@stitchdata.com \
+                     --password=$SANDBOX_PASSWORD \
+                     --token=$DEV_TOKEN \
+                     --client-id=50 \
+                     tests
```

Comment:
something like `pip install ../.[test]` prior to the `run-test` command where the test group contains the same paramiko 2.6 dependency that is required by the tap.

Reply:
Added the `test` group to install paramiko.